apm-protocol/apm-ui-protocol/src/main/resources/ui-graphql/trace.graphqls

```diff
@@ -42,9 +42,17 @@ input TraceQueryCondition {
   minTraceDuration: Int
   # The max time of trace
   maxTraceDuration: Int
+  state: TraceState!
+  order: QueryOrder!
```

Comment:
Why are the new condition mandatory？ Others are not like these.

Reply:
Because the default should be controlled by ui. it is not about mandatory, it is about explicit query